xtask/src/main.rs

```diff
@@ -0,0 +1,47 @@
+use std::{env, process::Command};
+
+use anyhow::Result;
+
+fn main() {
+    if let Err(e) = try_main() {
+        eprintln!("{}", e);
+        std::process::exit(-1);
+    }
+}
+
+fn try_main() -> Result<()> {
+    let task = env::args().nth(1);
+    match task.as_deref() {
+        Some("build") => build()?,
+        _ => print_help(),
+    }
+    Ok(())
+}
+
+fn print_help() {
+    eprintln!(
+        "Tasks:
+
+build            builds all languages 
+"
+    )
+}
+
+fn build() -> Result<()> {
+    run_libsql_sqlite3("./configure")?;
```

Comment:
We need a way to pass parameters here, some for configure, and some for make... Any easy way to pass them here?

What I mean is that I often build libsql like that:
```
make clean
./configure --enable-releasemode --enable-wasm-runtime
make liblibsql.la
```

Reply:
I don't know the different combinations and flags to pass through so not sure what makes sense, but I assume we can add them as needed. I just wanted to get a starter version of this going.